tests/test_dict_encoding.py

```diff
@@ -11,8 +11,8 @@
 import numpy as np
 import tskit
 
-import _msprime as c_module
```

Comment:
Let's change this to ``import msprime._msprime as c_module``. This'll make the diffs to the tskit version smaller and make our lives easier.

Reply:
Done!